integration/utils_test.go

```diff
@@ -33,14 +32,14 @@ func RunTests(t *testing.T, tests []GrizzlyTest) {
 				cwd, _ := os.Getwd()
 				cmd := exec.Command(filepath.Join(cwd, "../grr"), args...)
 				cmd.Dir = test.TestDir
-				output, err := cmd.CombinedOutput()
+				output, err := cmd.Output()
 				if command.ExpectedError != nil {
 					require.Error(t, err, command.ExpectedError)
 				}
 				if command.ExpectedOutput != "" {
 					data, err := os.ReadFile(filepath.Join(test.TestDir, command.ExpectedOutput))
-					require.NoError(t, err, fmt.Sprintf(""))
-					require.Contains(t, string(output), string(data))
```

Comment:
we could add `command.ExpectedContains` if we want the `Contains` functionality back.

Reply:
Yeah, it would be unused for now. We can add it when we need it, right?

Reply:
exactly